website/docs/quick-start/installation/modal/app.py

```diff
@@ -32,17 +32,21 @@ def download_model():
     .pip_install("asgi-proxy-lib")
 )
 
-stub = Stub("tabby-server-" + MODEL_ID.split("/")[-1], image=image)
+app = App("tabby-server-" + MODEL_ID.split("/")[-1], image=image)
 
+ee_volume = Volume.from_name("tabby-ee-vol", create_if_missing=True)
+ee_dir = "/data/ee"
 
-@stub.function(
+@app.function(
     gpu=GPU_CONFIG,
     allow_concurrent_inputs=10,
     container_idle_timeout=120,
     timeout=360,
+    volumes={ee_dir: ee_volume},
```

Comment:
add flag to ensure there'll be only one active process? (since ee dir expects have single writer)

Reply:
add concurrency_limit=1.